README.md

```diff
@@ -32,3 +32,51 @@ Out of the working groups, we anticipate recommendations on governance that will
+
+Second, the calls serve as Open Office Hours to reach community consensus on the foundation's next phases and to address questions and blockers on DevRel Challenges and Frequently Asked Questions.
+
+<details>
+  <summary>
+    <strong> 📚 How to File Proposals and Discussion Topics </strong>
+  </summary>
+
+The community can file proposals via the DRF Governance repo by opening a new entry in the GitHub Discussions Forum. Whether you are new to the DRF community calls or a regular participant, you can add your discussion topics as a comment below or bring them up during the call. People can also share DevRel use cases, best practices, frameworks, and more by opening a new issue in GitHub Discussions, which serves as the Foundation’s public forum. 
+
+We are always looking for 5-10 minute presentations in future community calls to spark discussion with attendees and asynchronously.
+
+</details>
+<details>
+  <summary>
+    <strong> 📚 Roles in the DRF Community Calls</strong>
+  </summary>
+
+Roles include facilitator, moderator, scribe, and crier. 
+
+Anyone can volunteer to take on any of these roles for upcoming community calls. People can volunteer to lead in one of these roles during the next call by contacting steering-committee@dev-rel.org
+
+
+| **Role**       | **Responsibilities**                                                                                                                                              |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Moderator**  | Leads the discussion by going through agenda items, ensuring each topic receives the time it needs, and keeping conversations focused. Volunteering as a Moderator is a great way to develop facilitation skills and play an essential role in shaping community discussions.|
+| **Scribe**     | Captures and summarizes key points during the call, sharing these notes with the community in the dedicated GitHub Discussion afterward. Serving as a Scribe builds attention to detail and provides a vital resource for transparent communication within the community.|
+| **Crier**      | Keeps the community informed by announcing the agenda in the community mailing list and Discord channel. As the Crier, you enhance community engagement and ensure members are aware of discussion topics and opportunities to participate.|
```

Comment:
The Moderator and Crier have always seemed very similar to me. And these definitions do suggest considerable overlap. Or am I missing something?

Reply:
There are a few differences, maybe I should be more specific in the description to make it clear:

- The Moderator must be on the call while the Crier can skip the call
- The Crier announces and encourages community participation _prior_ to the call starting and during the same week or the scheduled call. To put a clear example it would be that person who writes in Discord and Community list 

> "hey @ channel, ICYMI, the agenda is ready via GH discussions and google doc. Please don't forget to add your discussion topics, or open a new issue in GH discussions. 
> 
> Community calls are a neutral and safe space who follows chatham house rules and LF antitrust policy. The aim of these calls is to engage with the community and initiate conversations around proposals, challenges, questions, best practices, and more together.
> 
> Bellow are call details:
> 
> [Time] [Day] [Type]"

- The moderator is the one who is on the call at the scheduled time and hout to go through agenda topics. To make things more simple, the moderator also acts as the Facilitator, who opens the call, inviting participants to share their names in the agenda, and make group intros. it also introduces the concept of chathan house rules and informs about the DRF Code of conduct